mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -45,6 +45,28 @@ private[feature] trait RFormulaBase extends HasFeaturesCol with HasLabelCol {
  * Implements the transforms required for fitting a dataset against an R model formula. Currently
  * we support a limited subset of the R operators, including '~', '.', ':', '+', and '-'. Also see
  * the R formula docs here: http://stat.ethz.ch/R-manual/R-patched/library/stats/html/formula.html
+ *
+ * The basic operators are:
+ *
+ * `~` separate target and terms
```

Comment:
This is not the correct ScalaDoc syntax for a list. See https://wiki.scala-lang.org/display/SW/Syntax. You can run `build/sbt mllib/doc` to check the generated html API doc.